--- firebird.conf.original	2011-10-03 15:49:16.000000000 +0600
+++ firebird.conf.patched	2011-11-22 15:51:40.000000000 +0600
@@ -528,6 +528,7 @@
 #
 #RemoteServiceName = gds_db
 #RemoteServicePort = 3050
+RemoteServicePort = 3050
 
 #
 # The TCP Port Number to be used for server Event Notification
@@ -537,6 +538,7 @@
 # Type: integer
 #
 #RemoteAuxPort = 0
+RemoteAuxPort = 3060
 
 #
 # TCP/IP buffer size for send and receive buffers of both the client
